backend/templates/base.html

```diff
@@ -15,23 +15,30 @@
     <link rel="stylesheet" href="{% static 'css/bootstrap_4.4.1/bootstrap.min.css' %}"
           integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="{% static 'css/general.css' %}">
-    <link rel="apple-touch-icon" sizes="57x57" href="{% static 'img/favicon/apple-icon-57x57.png' %}">
-    <link rel="apple-touch-icon" sizes="60x60" href="{% static 'img/favicon/apple-icon-60x60.png' %}">
-    <link rel="apple-touch-icon" sizes="72x72" href="{% static 'img/favicon/apple-icon-72x72.png' %}">
-    <link rel="apple-touch-icon" sizes="76x76" href="{% static 'img/favicon/apple-icon-76x76.png' %}">
-    <link rel="apple-touch-icon" sizes="114x114" href="{% static 'img/favicon/apple-icon-114x114.png' %}">
-    <link rel="apple-touch-icon" sizes="120x120" href="{% static 'img/favicon/apple-icon-120x120.png' %}">
-    <link rel="apple-touch-icon" sizes="144x144" href="{% static 'img/favicon/apple-icon-144x144.png' %}">
-    <link rel="apple-touch-icon" sizes="152x152" href="{% static 'img/favicon/apple-icon-152x152.png' %}">
-    <link rel="apple-touch-icon" sizes="180x180" href="{% static 'img/favicon/apple-icon-180x180.png' %}">
-    <link rel="icon" type="image/png" sizes="192x192" href="{% static 'img/favicon/android-icon-192x192.png' %}">
```

Comment:
Android disappeared?

Reply:
Honestly, it takes a full SAP-team to do this right. It is a workable solution for now, and if you trust [this website](https://realfavicongenerator.net/favicon_checker?protocol=https&site=match4healthcare.de), there is at least some icon that Android understands. There is no easy python tool for this job, doing it manually is tedious and not worth its time since this would only be a placeholder icon for instances of our projects. I hope there are some good libraries for js and web stuff that do this right, repeatably, consistent and complete. Malte is looking into this. Until then, this should work for most scenarios.

Reply:
And btw, as you can see on the website, these links don't even work in m4h. There is some serious cleanup to be done and these lines have no function at the moment anyway I think.

Reply:
The website only finds the manifest and no icon. I think we used [https://favicon.io/favicon-converter/](https://favicon.io/favicon-converter/) before unless @andyz02 changed them in the meantime. [https://match4healthcare.de/static/img/favicon/android-icon-192x192.png](https://match4healthcare.de/static/img/favicon/android-icon-192x192.png) works for me.

Reply:
I believe that some of the files need to be available from the root dir, which won't work with the current statics setup :(
https://realfavicongenerator.net/ (mentioned by @Baschdl ) seems to be good. But I think we need to research this, whether it's possible (if even necessary) to make some of those files reachable from the app root. 

We _could_ always redirect or serve them with nginx but we **probably shouldn't** if at all possible.

Reply:
I think one way around the is to include lots of meta links in the html base header who link to the appropriate places, lots of favicon generators generate that html for you directly, so this should work. However, the current setup was and is flawed and this needs to be cleaned up nicely. I think a good moment for that would be when we switch to the automatic approach with a JS library that we envisioned @maltezacharias, right?

Reply:
Yep. is there an issue for that? If not please create one. And for now we can merge this.

Reply:
I set it up with a redirect in the very beginning of our project but I don't think we need it if we have the meta links. The 404 for `favicon.ico` shouldn't be a problem because the browser finds the appropriate icons with the meta links, see https://stackoverflow.com/a/59715513/3342058